src/pages/TransactionPage/index.tsx

```diff
@@ -45,6 +45,14 @@ export function TransactionPage() {
   const { data } = useTransactionPageQuery({ variables: { id: transaction } });
   const tx = data?.transaction;
 
+  function thousandSeparator(number : any) {
```

Comment:
Both `tx.gasPrice` and `tx.gasLimit` are `string`, i think this can be type `string`

Reply:
Thank you, will try type string.